solutions/usaco-762.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-762
+source: USACO Silver 2017 December
+title: My Cow Ate My Homework
+author: Juheon Rhee
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_homework_silver_dec17.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+with open('homework.in') as read:
+        n = int(read.readline())
+        questions = [*map(int, read.readline().split())]
+
+left, right = -1, n
+total, minimum, ans = [0] * n, [0] * n, []
```

Comment:
(optional) better name for `ans`
what's the answer in the context of the problem?

Reply:
values to obtain maximum score, so i changed it to values.

Reply:
`max_score_values`?

Reply:
alright